src/styles/_extends.scss

```diff
@@ -0,0 +1,7 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
```

Comment:
I would suggest renaming this file. Maybe "normalize.scss" could be better?:)
The underscore "_" itself is used for utilies files. For instance, "_variables.scss", "_mixins.scss" and etc.